ruleset/rules/0630-nextcloud_rules.xml

```diff
@@ -1,175 +1,156 @@
 <!--
-  -  NextCloud rules
-  -  Author: banditopazzo.
-  -  Updated by Wazuh, Inc.
-  -  Copyright (C) 2015-2021, Wazuh Inc.
-  -  This program is a free software; you can redistribute it and/or modify it under the terms of GPLv2.
+  Copyright (C) 2015-2021, Wazuh Inc.
 -->
 
 <!--
-- It's needed to enable audit in Nextcloud if you want full log
-- It's needed to paste the following in /var/ossec/etc/ossec.conf for the decoder to work properly:
```

Comment:
This line is confusing, since it talks about decoders and a configuration file. A user might assume that this line necessarily goes on the manager. Is this true, does this go on the manager or agent? Does the AWS integration need to be configured somewhere else? What are these variables:
- ABSOLUTE_PATH_TO_YOUR_AUDIT
- ABSOLUTE_PATH_TO_YOUR_NEXTCLOUD